datajoint_utilities/dj_search/lists.py

```diff
@@ -1,11 +1,11 @@
 import datajoint as dj
-from pymysql import OperationalError
+from pymysql import IntegrityError, OperationalError
 
 """
 Development helper functions. Chris Brozdowski <CBroz@datajoint.com>
```

Comment:
we don't need to put the authors in the code. The code version control system keeps track of everyone's contributions.

Reply:
Sure, I can remove this in a PR to @iamamutt's fork

Reply:
@CBroz1 @dimitri-yatsenko merged PR